tools/wsdlto/databinding/jaxb/src/main/java/org/apache/cxf/tools/wsdlto/databinding/jaxb/JAXBDataBinding.java

```diff
@@ -437,24 +437,20 @@ public void initialize(ToolContext c) throws ToolException {
                 throw new ToolException(e);
             }
         }
-        addSchemas(opts, schemaCompiler, schemas);
-        addBindingFiles(opts, jaxbBindings, schemas);
-
 
-        for (String ns : context.getNamespacePackageMap().keySet()) {
-            File file = JAXBUtils.getPackageMappingSchemaBindingFile(ns, context.mapPackageName(ns));
+        if (context.optionSet(ToolConstants.CFG_SUPPRESS_GENERATED_DATE)) {
+            // Prevents dumping current date as part of javadocs of the Java files generated.
+            // This is done by passing '-suppress-generated-date' attribute to jaxb xjc.
```

Comment:
XJC has no such option, why should we pass it? After removing this codeblock the test still passes.

Reply:
fixed, thanks